blocks/right-nav/right-nav.css

```diff
@@ -0,0 +1,26 @@
+/* Right Nav CSS */
+
+@media only screen and (min-width: 1200px) {
+  main .section.right-nav-container > div {
+    width: unset;
+  }
+
+  .section.right-nav-container {
+    position: relative;
+    display: grid;
+    grid-template-columns: 3fr 1fr;
+    width: var(--section-width-desktop);
+    margin: 0 auto;
+  }
+
+  .section.right-nav-container > *:not(.right-nav-wrapper) {
+    grid-column-start: 1;
+  }
+
+  .section.right-nav-container > .right-nav-wrapper {
+    grid-column-start: 2;
+    grid-row-start: 1;
+    padding-top: var(--spacer-layout-07);
+    margin-left: 8%;
+  }
+}
```

Comment:
Nitpicking: I think we should rename this block `right-nav` to `sidebar`. "Right Nav" implies that it's navigational content, however, this is more supplemental info.

Reply:
feel free to change the name. whatever makes more sense for you